meta/main.yml

```diff
@@ -1,5 +0,0 @@
----
-allow_duplicates: yes
```

Comment:
allow_duplicates seems unrelated to the role dependency, should possibly be kept?